content/en/providers/check-in/sp/_index.md

```diff
@@ -671,7 +673,9 @@ user authorisation:
 | Description                                                                                     | OIDC Claim              |
 | ----------------------------------------------------------------------------------------------- | ----------------------- |
 | [VO/group membership/roles of the authenticated user](#vogroup-membership-and-role-information) | `eduperson_entitlement` |
-| [Level of Assurance (LoA)](#level-of-assurance)                                                 | `acr`                   |
+| [Capabilities](#capabilities)                                                                   | `eduperson_entitlement` |
+| [GOCDB roles](#gocdb-roles)                                                                     | `eduperson_entitlement` |
+| [Level of Assurance (LoA)](#level-of-assurance)                                                 | `eduperson_assurance`   |
```

Comment:
Level of Assurance --> Identity Assurance

Reply:
Done